rakefile

```diff
@@ -74,9 +74,9 @@ def configure_opsworks
 end
 
 desc 'Provision systems for the performance task with ABS - included in the performance task'
-rototiller_task :performance_provision_with_abs do |t|
-  raise "The BEAKER_INSTALL_TYPE env var must be set to either 'pe' or 'foss' to continue" unless
-      ENV['BEAKER_INSTALL_TYPE'] == 'pe' || ENV['BEAKER_INSTALL_TYPE'] == 'foss'
+rototiller_task :performance_provision_with_abs  => "validate:verify_beaker_install_type" do |t|
```

Comment:
I believe you said this had to happen here to get in front of pe_xl doing the install.  Since it is not intuitive to think this needs to come before provisioning add a comment explaining that.

Reply:
I've added the following comment: `# validate the BEAKER_INSTALL_TYPE (and dependent env vars) to prevent provisioning if incorrectly specified`